library/src/main/java/com/mikepenz/materialdrawer/model/ProfileSettingDrawerItem.java

```diff
@@ -89,7 +91,7 @@ public ProfileSettingDrawerItem withName(String name) {
     }
 
     public ProfileSettingDrawerItem withDescription(String description) {
-        this.email = new StringHolder(description);
+        this.description = new StringHolder(description);
```

Comment:
I want to keep Email seperate for different implementations. Let me know if it has an issue.